tests/cxx/implicit_ctor.cc

```diff
@@ -10,18 +10,30 @@
 // IWYU_ARGS: -Xiwyu --check_also="tests/cxx/*-d1.h" -I .
 
 // If you define an API that accepts an argument of class type or
-// const reference to a class type with an implicit constructor, you
-// must provide the definition for the class.
+// const reference to a class type with an implicit constructor which takes
+// exactly one argument, you must provide the definition for the class.
 //
-// If you use an API and that use triggers an implicit constructor
+// If you use an API and that use triggers such an implicit constructor
 // call, you are *not* considered to have used that constructor,
 // relying on the previous rule to make the code complete definition
-// available.
+// available. This is so called "autocast".
 //
 // This tests that logic.
 
 #include "tests/cxx/implicit_ctor-d1.h"
 
+// No 'autocast' assurance for .cpp-file-local functions...
+// IWYU: IndirectWithImplicitCtor needs a declaration
+static int LocalFn(const IndirectWithImplicitCtor&) {
+  return 1;
+}
+
+// ... and for .cpp-file-definitions of functions declared in a header.
```

Comment:
s/and/or/

Reply:
Sorry?

Reply:
Oh, sorry! That's the sed (and vim, I believe) command sequence for search-and-replace:
https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html

I really should have said "replace 'and' with 'or' here", but my fingers outran my empathetic brain :)